hugegraph-common/src/main/java/com/baidu/hugegraph/config/HugeConfig.java

```diff
@@ -45,7 +45,7 @@
 
     private static final Logger LOG = Log.logger(HugeConfig.class);
 
-    private final String path;
+    private String path;
 
     public HugeConfig(Configuration config) {
```

Comment:
can extract file from Configuration as possible?

Reply:
I dont have idea, the ProperityConfiguration have a private field 'Location', but not get method

Reply:
For the convenience of users, we can try to use a trick way like Whitebox